lightning-invoice/src/utils.rs

```diff
@@ -186,8 +186,8 @@ where
 		)
 		.map_err(|_| SignOrCreationError::CreationError(CreationError::InvalidAmount))?
 	};
-
-	log_trace!(logger, "Creating phantom invoice from {} participating nodes with payment hash {}",
+	// todo: example of external crate using log_trace! initially now unable to re-export with pub use _log_trace_with_peer_id_channel_id as log_trace
```

Comment:
an issue with this approach is that re-exporting within an external crate isnt possible (compiler seems to disagree) and also feels very wierd.

seems like Rust doesnt allow / like macros to be overloaded